receiver/splunkenterprisereceiver/metadata.yaml

```diff
@@ -12,6 +12,12 @@ attributes:
   splunk.host:
     description: The name of the splunk host
     type: string 
+  splunk.splunkd.build:
+    description: The build number for this Splunk instance version
+    type: string
+  splunk.splunkd.version:
+    description: The splunkd version number
```

Comment:
why are these not resource attributes?

Reply:
very fair question. the reason why not is because not every metric (necessarily) pulls from the same splunkd instance. for example: if you're monitoring metrics from a search head as well as an indexer these may be completely separate splunk deployments.

ideally we would have a way ([upstream](https://github.com/open-telemetry/opentelemetry-collector/issues/10726)) to optionally set attributes in a way which would behave like resource attributes.